packages/DApp/src/components/card/VoteConfirmModal.tsx

```diff
@@ -22,7 +22,8 @@ export function VoteConfirmModal({ community, selectedVote, setShowModal }: Vote
         <span>
           {selectedVote.verb} {community.name}
         </span>{' '}
-        will be casted after 10 confirmations.
+        {/* will be casted after 10 confirmations. */}
```

Comment:
This can be removed altogether. Votes are not interacting with a blockchain whatsoever.

Reply:
Removed the "cast" keyword references since they might connote with `castVote` contract method.